metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -130,10 +130,10 @@ object SbtBuildTool {
       ""
     }
     s"""|// DO NOT EDIT! This file is auto-generated.
-        |// This file enables sbt-bloop to create bloop config files.
-        |$resolvers
-        |addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "$bloopSbtVersion")
-        |""".stripMargin
+       |// This file enables sbt-bloop to create bloop config files.
```

Comment:
Should this be changed after adding the align setting?

Reply:
Yea there still seems to be some formatting on multilines happening after the addition of the align setting. I'll look at this week and ensure everything is working fine and address the points above. Plus, some of the formatting on the multilines are also breaking tests.